x/smart-account/authenticator/authentication_request.go

```diff
@@ -266,14 +234,32 @@ func GenerateAuthenticationRequest(
 		MsgIndex:   uint64(msgIndex),
 		Signature:  msgSignature,
 		TxData:     txData,
-		SignModeTxData: SignModeData{ // TODO: Add other sign modes. Specifically textual when it becomes available
-			Direct: signBytes,
+		SignModeTxData: SignModeData{
+			Direct: []byte("signBytes"),
```

Comment:
Is there a reason to have this instead of nil?

Reply:
Originally had this nil, but cosmwasm authenticators will fail because of the interface